### PR TITLE
style-navbar styling

### PR DESCRIPTION
**Pull request by ImrozNezami**
Nav-bar

Screenshot (227)

### CODE REVIEW

File: nav-bar.css [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+}
+.navbar-icons {
+  display: flex;
+  gap: 2.2rem;
+}
+.nav-inp {
+  padding: 0.6rem;
+  width: 20%;
+  outline: 2px solid transparent;
+  outline-offset: 2px;
```
Comment (kjais1720) [inline nav-bar.css:20]: you can extract this in a common class since it will be used for almost every button on your webpage

File: nav-bar.css [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  border: 2px solid;
+  border-radius: 0.4rem;
+}
+.nav-title {
+  margin-left: 0.8rem;
+}
+.nav-btn {
+  outline: 2px solid transparent;
+  outline-offset: 2px;
```
Comment (kjais1720) [inline nav-bar.css:29]: same as above

File: nav-bar.css [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  left: 1.2rem;
+  border-radius: 50%;
+  color: var(--white);
+  font-size: 0.9rem;
+  width: 1.2rem;
+  height: 1.2rem;
+  text-align: center;
+  background-color: var(--alertred);
+  font-weight: bold;
+}
```
Comment (Tanjum786) [inline nav-bar.css:54]: Use beautify file it maintains code in discipline. other than all is good.

**Review by Tanjum786 — APPROVED** [review]

File: nav-bar.css (hunk 3 above)
Comment (ImrozNezami) [reply nav-bar.css:54]: Okay will keep in mind.

File: nav-bar.css (hunk 1 above)
Comment (ImrozNezami) [reply nav-bar.css:20]: Okay

**Pull request merged**